Ginger/Ginger/Actions/ActionConversion/ActionsConversionWizard.cs

```diff
@@ -218,16 +223,33 @@ public async void BusinessFlowsActionsConversion(ObservableList<BusinessFlowToCo
             }
             finally
             {
-                Reporter.HideStatusMessage();               
+                Reporter.HideStatusMessage();
             }
         }
-        
+
         /// <summary>
-        /// This method is used to cancle the wizard
+        /// This method will restore the conversion done
         /// </summary>
         public override void Cancel()
         {
-            base.Cancel();
+            try
+            {
+                if (IsConversionDoneOnce)
+                {
+                    foreach (BusinessFlowToConvert bfToConvert in ListOfBusinessFlow)
```

Comment:
try-Finally should be done as you did to insure the base.Cancel but try-catch for restore failure exception should be done around the single bfToConvert.BusinessFlow.RestoreFromBackup(); so if one will fail we still will try to restore others

Reply:
Done